### PR TITLE
chore(build): update linter configuration

### LINKED ISSUE #227: Update linter configuration
The eslint configuration allows for too many inconsistencies in the code at the moment. We need to update the eslint config to define more strict rules on;
* semicolons (rule: semi)
* empty line at end (rule: eol-last)
* brace style (rule: brace-style)

A single definition for all packages
Check only src or also test ?
And what about custom rules? For example start of file should contain emtpy line?

### PR DESCRIPTION
**Pull request by BasMasking**
Fixes #227 

Changes proposed in this pull request:
- One single configuration for all packages
- Added custom rule for empty first lines
- Added new rules
- Fixed all issues generated by the new configuration

@MaskingTechnology/jitar

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,6 +13,7 @@
         "packages/runtime",
         "packages/serialization",
         "packages/server-nodejs",
+        "eslint-plugin",
```
Comment (petermasking) [inline package.json:16]: Should we place this plugin in a folder for all build tools/scripts? This might help keeping the root clean

File: .eslintrc.json
Comment (petermasking) [inline .eslintrc.json:1]: Is it an idea to exclude the create-jitar/template folder? Now the templates have all semicolumns, which is great but deviates from the templates provided by Vite. Do we want this?

**Review by petermasking — CHANGES_REQUESTED** [review]
Looks great! Added two questions to this review.

File: package.json (hunk 1 above)
Comment (BasMasking) [reply package.json:16]: a _tools_ folder in the root? So the sturcture becomes tools/eslint-plugin?

File: .eslintrc.json
Comment (BasMasking) [reply .eslintrc.json:1]: I don't know if we want this. The custom code we've added is now added according to our own standards. I think it's best to keep Vite and custom code synced, but I don't have a preference for either one.

**Review by petermasking — DISMISSED** [review]

**Review by petermasking — APPROVED** [review]

**Pull request merged**